src/Views/WhatsNewPage.xaml.cs

```diff
@@ -21,6 +22,12 @@ namespace DevHome.Views;
 
 public sealed partial class WhatsNewPage : Page
 {
+    public Uri DevDrivePageKeyUri { get; private set; } = new ("ms-settings:disksandvolumes");

```

Comment:
If I'm understanding correctly, you're setting the link to this, then if it is set to this but it's not available, you change it?
I think it would be better to just set it right the first time

Reply:
not in this case? So how it works in the xaml looks like each card has a page ID associated with it. That is why, see here: https://github.com/microsoft/devhome/blob/5f099b1911239490fe9fb23f597d1eab7052cd20/src/Views/WhatsNewPage.xaml#L115 

I didn't want to change the behavior too much with this change. I don't change this variable, what I'm changing is the localized text string  from "Create  Dev Drive" to "Learn more" 

https://github.com/microsoft/devhome/blob/5f099b1911239490fe9fb23f597d1eab7052cd20/src/Views/WhatsNewPage.xaml#L197

Reply:
spoke offlline about this, we'll be refactoring this page so will leave as is for now